azure-pipeline-templates/verbose-tests.yml

```diff
@@ -112,6 +119,25 @@ steps:
   - script: cat ${{ parameters.spn_credential_config }}
     displayName: Print SPN config file
 
+  # Create azurite config file if we need to test it
+  - script: |
+      cd ${{ parameters.working_dir }}
+      ${{ parameters.working_dir }}/blobfuse2 gen-test-config --config-file=azure_key.yaml --container-name=${{ parameters.container }} --temp-path=${{ parameters.temp_dir }} --output-file=${{ parameters.azurite_config }}
+    displayName: Create Key Config File
+    # Note: It is fine to hardcode these credentials since they are publicly documented fake credentials for Azurite
+    env:
+      ACCOUNT_TYPE: ${{ parameters.account_type }}
+      NIGHTLY_STO_ACC_NAME: 'devstoreaccount1'
+      NIGHTLY_STO_ACC_KEY: 'Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
+      ACCOUNT_ENDPOINT: 'http://127.0.0.1:10000/devstoreaccount1'
+      USE_HTTP: 'true'
```

Comment:
ok I see its this place where we send as true. does that mean azurite test is only over http and not https ?

Reply:
yeah. They have support for https, but you need to turn some other flags on and pass in a ca cert and stuff, so it seemed not necessary